chsdi/locale/de/LC_MESSAGES/chsdi.po

```diff
@@ -4250,11 +4352,13 @@ msgstr "Punktnummer"
 msgid "ch.swisstopo.geologie-gravimetrischer_atlas.metadata"
 msgstr "Gravimetrischer Atlas 100 - Papier"
 
+#. TODO
 msgid "ch.swisstopo.geologie-gravimetrischer_atlas.metadata.id"
-msgstr "Nr."
+msgstr "ch.swisstopo.geologie-gravimetrischer_atlas.metadata.id"
```

Comment:
Why do we keep loosing those? I added them to the transaltion table just before to leave. @AFoletti Can you try to make translate again please?